lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -24,13 +24,14 @@
 #include "glow/IR/Instrs.h"
 #include "glow/LLVMIRCodeGen/LLVMIRGen.h"
 #include "glow/Support/Debug.h"
+#include "glow/Support/Register.h"
```

Comment:
Why do we need this include here? Where is it used?

Reply:
Should not be here, it was an artifact of having registration in this file.